x-pack/test/functional/services/uptime/navigation.ts

```diff
@@ -30,17 +40,29 @@ export function UptimeNavigationProvider({ getService, getPageObjects }: FtrProv
       await testSubjects.existOrFail('uptimeSettingsPage', { timeout: 2000 });
     },
 
-    goToMonitor: async (monitorId: string, monitorName?: string) => {
-      await testSubjects.click(`monitor-page-link-${monitorId}`, 5000);
-      if (
-        monitorName &&
-        (await testSubjects.getVisibleText('monitor-page-title')) !== monitorName
-      ) {
-        throw new Error('Expected monitor name not found');
+    checkIfOnMonitorPage: async (monitorId?: string) => {
```

Comment:
```suggestion
    checkIfOnMonitorPage: async (monitorId: string) => {
```

Any reason we can't require this parameter?